src/sparseml/export/export.py

```diff
@@ -45,7 +45,6 @@ def export(
     opset: int = TORCH_DEFAULT_ONNX_OPSET,
     single_graph_file: bool = True,
     num_export_samples: int = 0,
-    batch_size: int = 1,
```

Comment:
Removing batch_size argument from the export.
By convention, all our sample inputs/outputs/labeled are stored in the "batchless" arrays, e.g. inp-0000.npz has shape `(3, 244, 244)`